### PR TITLE
Louisiana adoption credit

### LINKED ISSUE #3693: Louisiana refundable credit for the adoption of unrelated infant
ref: 
https://revenue.louisiana.gov/TaxForms/IT540i%20WEB(2023)%20INSTRUCTIONS.pdf

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #3693

### CODE REVIEW

File: policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/amount.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+description: Louisiana provides this amount of refundable credit for the adoption of a child.
+values:
+  2023-01-01: 5000
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/amount.yaml:3]:
```suggestion
  2023-01-01: 5_000
```

File: policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/amount.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+ period: year
+ unit: year
+ label: Louisiana adoption tax credit
+ reference:
+ - title: R.S. 47:297.23 
+   href: https://legis.la.gov/legis/Law.aspx?d=1336834
+ - title: ACT NO.452
+   href: https://legis.la.gov/legis/ViewDocument.aspx?d=1333450
+ - title: General Information for Filing Your 2023 Louisiana Resident Individual Income Tax Return
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/amount.yaml:14]:
```suggestion
 # Credit was introduces in 2023
 - title: General Information for Filing Your 2023 Louisiana Resident Individual Income Tax Return
```

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/amount.yaml:14]: add to all yaml files

File: policyengine_us/variables/gov/states/la/tax/income/credits/adoption/la_adoption_credit.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/la/tax/income/credits/adoption/la_adoption_credit.py:1]: add to list of refundable credits in la

File: policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/age_threshold.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+description: Louisiana provides a refundable credit for the adoption of a child below this age threshold.
+values:
+  2023-01-01: 2
+
+metadata:
+ period: year
+ unit: year
+ label: Louisiana adoption tax credit child age threshold
+ reference:
+ - title: R.S. 47:297.23
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/age_threshold.yaml:10]: add section if applicable

File: changelog_entry.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - add 2023 louisiana adoption refundable credit.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - Louisiana adoption credit.
```

File: policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/age_threshold.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+description: Louisiana provides a refundable credit for the adoption of a child below this age threshold.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/age_threshold.yaml:1]:
```suggestion
description: Louisiana provides the adoption credit for each adopted child below this age threshold.
```

File: policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/age_threshold.yaml [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+values:
+  2023-01-01: 2
+
+metadata:
+ period: year
+ unit: year
+ label: Louisiana adoption tax credit child age threshold
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/age_threshold.yaml:8]:
```suggestion
 label: Louisiana adoption credit child age threshold
```

File: policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/amount.yaml [hunk 7]
```diff
@@ -0,0 +1,16 @@
+description: Louisiana provides this amount of refundable credit for the adoption of a child.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/amount.yaml:1]:
```suggestion
description: Louisiana provides an adoption credit of this amount for each eligible child.
```

File: policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/amount.yaml [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+values:
+  2023-01-01: 5_000  
+
+metadata:
+ period: year
+ unit: year
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/adoption/amount.yaml:7]:
```suggestion
 unit: currency-USD
```

File: policyengine_us/parameters/gov/states/la/tax/income/credits/refundable.yaml [hunk 9]
```diff
@@ -0,0 +1,7 @@
+description: Louisiana refundable tax credits.
+metadata:
+  unit: list
+  label: Louisiana refundable credits
+values:
+  2023-01-01:
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/refundable.yaml:6]:
```suggestion
  2021-01-01: []
  2023-01-01:
```

File: policyengine_us/parameters/gov/states/la/tax/income/credits/refundable.yaml [hunk 10]
```diff
@@ -0,0 +1,7 @@
+description: Louisiana refundable tax credits.
+metadata:
+  unit: list
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/refundable.yaml:3]:
```suggestion
  unit: list
  period: year
```

File: policyengine_us/parameters/gov/states/la/tax/income/credits/refundable.yaml [hunk 11]
```diff
@@ -0,0 +1,7 @@
+description: Louisiana refundable tax credits.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/refundable.yaml:1]:
```suggestion
description: Louisiana provides the following refundable tax credits.
```

File: policyengine_us/parameters/gov/states/la/tax/income/credits/refundable.yaml [hunk 12]
```diff
@@ -0,0 +1,7 @@
...
+metadata:
+  unit: list
+  label: Louisiana refundable credits
+values:
+  2023-01-01:
+    - la_adoption_credit
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/la/tax/income/credits/refundable.yaml:7]: add reference (for now just adoption credit law reference)

File: policyengine_us/tests/policy/baseline/gov/states/la/tax/income/credits/adoption/la_adoption_credit_person.yaml [hunk 13]
```diff
@@ -0,0 +1,51 @@
+- name: In 2023, taxpayer adopted 1 children, age >2, the louisiana adoption credit was 0
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/la/tax/income/credits/adoption/la_adoption_credit_person.yaml:1]:
```suggestion
- name: In 2023, taxpayer adopted 1 child, age >2, the louisiana adoption credit was 0
```

Comment (PavelMakarchuk) [reply policyengine_us/tests/policy/baseline/gov/states/la/tax/income/credits/adoption/la_adoption_credit_person.yaml:1]: and others

File: policyengine_us/data/datasets/cps/cps.py [hunk 14]
```diff
@@ -448,34 +448,57 @@ def add_previous_year_income(self, cps: h5py.File) -> None:
...
+    )
+    df["self_employment_income_prev"] = np.ones(len(df)) * np.nan
+    df["self_employment_income_prev"][cps_cur_record_in_sample] = (
+        cps_previous_year.loc[
+            cps_current_year.index[cps_cur_record_in_sample]
+        ].SEMP_VAL.values
+    )
+
+    X = cps_current_year[PREDICTORS][~cps_cur_record_in_sample]
+    X = X.rename(columns={x: x + "_cur" for x in PREDICTORS})
```
[the remark this answers is not in the document]
Comment (yy3187) [reply policyengine_us/data/datasets/cps/cps.py:490]: i just run it, and there is nothing changed.

**Pull request merged**